docs/pages/machine-id/getting-started.mdx

```diff
@@ -0,0 +1,244 @@
+editor                                                            user:list,create,read,update,delete,...
+```
+
+Machine ID can join with a token or the [IAM Method](https://goteleport.com/docs/setup/guides/joining-nodes-aws) on AWS.
+
+<Tabs>
+  <TabItem label="Token-based Joining">
+  ```code
+  $ tctl bots add robot --roles=access
+  ```
+  </TabItem>
+  <TabItem label="IAM Method">
+  First, create an IAM method token that specifies the AWS account from which
+  the bot can join.
+
+  ```
+  kind: token
+  version: v2
+  metadata:
+    # The token name is not a secret because instances must prove that they are
+    # running in your AWS account to use this token.
+    name: iam-token
+    # Set a long expiry time. The default for tokens is only 30 minutes.
+    expires: "3000-01-01T00:00:00Z"
+  spec:
+    # Only allow bots to join using this token.
+    roles: [Bot]
+  
+    # Set the join method to be IAM.
+    join_method: iam
+
+    # Define the name of the bot that will be allowed to use this token. Note,
+    # the "bot-" prefix is required. The name of the bot you will create is "robot".
+    bot_name: bot-robot
+  
+    allow:
+    # Restrict the AWS account and (optionally) ARN that can use this token.
+    # This information can be obtained from running the
+    # "aws sts get-caller-identity" command from the CLI.
+    - aws_account: "111111111111"
+      aws_arn: "arn:aws:sts::111111111111:assumed-role/teleport-bot-role/i-*"
+  ```
+
+  Next, create the bot user.
+
+  ```
+  $ tctl bots add robot --token=iam-token --roles=access
+  ```
+  </TabItem>
+</Tabs>
+
+## Step 3/4. Start Machine ID
+
+Now start Machine ID using the `tbot` binary. The `tbot start` command will
+start running Machine ID in a loop, writing renewable certificates to
+`/var/lib/teleport` and the short-lived certificates your application will use
+to `/opt/machine-id`.
+
+In a production environment you will want to run Machine ID in the background
+using a service manager like systemd. However, in this guide you will run it in
+the foreground to better understand how it works.
+
+<Tabs>
+  <TabItem label="Token-based Joining">
+  ```code
+  $ tbot start \
+     --data-dir=/var/lib/teleport \
+     --destination-dir=/opt/machine-id \
+     --token=00000000000000000000000000000000 \
+     --join-method=token \
```

Comment:
Note, when trying I get this error. `ERROR: unknown long flag '--join-method'`

Reply:
Strange, not seeing this issue, but I am running on `branch/v9` instead of beta.1.

```
$ tbot start --help
usage: tbot start [<flags>]

Starts the renewal bot, writing certificates to the data dir at a set interval.

Flags:
  -d, --debug            Verbose logging to stdout
  -c, --config           tbot.yaml path
  -a, --auth-server      Specify the Teleport auth server host
      --token            A bot join token, if attempting to onboard a new bot; used on first connect.
      --ca-pin           A repeatable auth server CA hash to pin; used on first connect.
      --data-dir         Directory to store internal bot data.
      --destination-dir  Directory to write generated certificates
      --certificate-ttl  TTL of generated certificates
      --renew-interval   Interval at which certificates are renewed; must be less than the certificate TTL.
      --join-method      Method to use to join the cluster.
```